CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.1] - 2017-01-10
```

Comment:
The formatting could use some love too. The first bullet under added and the first bullet under changed both have a leading `**` without any closing `**` which doesn't bold like it is meant to.